docs/sidebar.yml

```diff
@@ -159,7 +159,7 @@ subsection:
           - page: reference/language-versions/source-compatibility.md
           - page: reference/language-versions/binary-compatibility.md
       - page: reference/soft-modifier.md
-      - page: reference/features-classification.md
+      # - page: reference/features-classification.md  outdated copy of reference/index.html
```

Comment:
actually this is much newer than reference/index.html, and the purpose is different, i.e. at the bottom of `reference/index.html` you see


> [A classification of proposed language features](http://dotty.epfl.ch/docs/reference/features-classification.html) is an expanded version of this page that adds the status (i.e. relative importance to be a part of Scala 3, and relative urgency when to decide this) and expected migration cost of each language construct.

Reply:
```suggestion
      - page: reference/features-classification.md
```